pywinauto/application.py

```diff
@@ -1135,24 +1156,24 @@ def __getattribute__(self, attr_name):
         return self[attr_name]
 
     def WriteAppData(self, filename):

```

Comment:
Maybe it's time to convert it to non-public method: ```_write_app_data```. Same for the GetMatchHistory method.